### ISSUE TITLE
Add object "Service" for a Host

### ISSUE THREAD

**Issue by opencommits** [T-Systems-MMS/ansible-collection-icinga-director#42]
There is no way to create a Director object "Service" (Individual Service object) for a single Host or can you give a example how to create this via the ansible role:

```
object Service "Webserver SSL" {
    host_name = "LINUXHOST10"
    import "servicetmpl_checkperiod_6-21x5"
    import "service_http-ssl-cert"
    vars.http_port = "443"
}
```

Thank you very much. Great work. :)

**Comment by rndmh3ro** [comment]
Hi @opencommits,

you're right - there is no possibility yet to create Service objects. I have some code written for this in this PR: https://github.com/T-Systems-MMS/ansible-collection-icinga-director/pull/32

However its not working yet.

**Comment by opencommits** [comment]
Are there any news regarding this issue to create a object of type 'Service'?

**Comment by rndmh3ro** [comment]
I updated the PR #32. It's getting reviewed.

**Comment by rndmh3ro** [comment]
The new module is released in v1.6.0: https://galaxy.ansible.com/t_systems_mms/icinga_director

Can you please test and give feedback? If there are any issues, please open a new issue.
